src/modules/app/components/Header.tsx

```diff
@@ -58,9 +59,49 @@ export class Header extends React.Component<Props> {
         this.handleClose();
     }
 
+    profile(user) {
+        const { anchorEl } = this.state;
+
+        switch(user) {
+            case undefined:
+                return <div>Loading...</div>; // TODO: Replace this with CircularProgress
```

Comment:
I couldn't get CircularProgress to work for the life of me, so I'm hoping someone else can lol